backstop/scenarios/surface.js

```diff
@@ -0,0 +1,19 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+const { scenario } = require('../scenarioHelper');
+
+module.exports = [
+  scenario('Type default', {
+    selectors: ['#demo-default'],
+    viewports: [{ width: 800, height: 600 }],
+    hideSelectors: ['h1', 'hr', '#demo-elevations'],
```

Comment:
Do we really need these? Since we are targeting specific sections, these elements are not even being captured.

Reply:
The negative margin & positive padding on the `<section>` to help capture the drop shadow are capturing some of the surrounding elements.  Screenshot of the test results after removing the `hideSelectors`:
![Screen Shot 2022-03-21 at 12 55 01 PM](https://user-images.githubusercontent.com/849817/159318527-86ede11d-a6fb-4a7d-bd5e-3028a56415e3.png)